css/style.css

```diff
@@ -0,0 +1,461 @@
+  padding: 2rem;
+  margin-top: 50px;
+  margin-bottom: 50px;
+  background-color: white;
+  border: 2px solid #e5e5e5;
+  list-style-type: none;
+  border-radius: 24px;
+}
+
+.work-container > ul {
+  display: grid;
+  list-style-type: none;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  grid-gap: 1rem;
+}
+
+.work-container > ul > li {
+  border-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  position: relative;
+}
+
+.work-container > ul > li > img {
+  width: 100%;
+}
+
+.work-container > ul > li > p {
+  font-size: 1rem;
+  line-height: 1.5;
+  padding: 1.2rem 0;
+  color: #666;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(620px, 1fr));
+  align-items: center;
+  grid-gap: 10px;
+  padding: 10px;
+  border: 0;
+  background: transparent;
+}
+
+.work-info ul li {
+  display: inline;
+  list-style: none;
+  padding-bottom: 30px;
+  padding-right: 7px;
+  color: #c1c7d0;
+  font-size: 20px;
+}
+
+.work-info ul li i {
+  font-size: 10px;
+}
+
+.intro {
+  text-transform: uppercase;
+}
+
+.tags ul li {
+  display: inline;
+  align-items: center;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 49px;
+  height: 20px;
+  font-size: 15px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.btn-one {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 4%;
+  font-size: 17px;
+  font-weight: 300;
+  text-transform: capitalize;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.btn-two {
+  border: 1px solid #4053fc;
+  background: #4053fc;
+  padding: 7px;
+  border-radius: 4%;
+  font-size: 17px;
+  font-weight: 300;
+  text-transform: capitalize;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+a.btn-one {
+  text-decoration: none;
+  color: #4053fc;
+}
+
+a.btn-two {
+  color: #fff;
+  text-decoration: none;
+}
+
+.btn-tags {
+  margin-bottom: 30px;
+}
+
+.section-about {
+  background-color: white;
+  border-top-right-radius: 100px;
+}
+
+.section-about h1 {
+  position: static;
+  width: 920px;
+  height: 52px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 35px;
+  letter-spacing: 0.37px;
+  color: #091e42;
+  margin: 40px 10px 30px 0;
+  font-size: 35px;
+}
+
+.about-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(420px, 1fr));
+  grid-gap: 20px;
+  align-items: center;
+}
+
+.about-grid > article {
+  padding-bottom: 50px;
+}
+
+.img-b {
+  margin-bottom: 250px;
+}
+
+.form {
+  position: relative;
+  bottom: 900px;
+}
+
+.form-text {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  color: #fff;
+  position: relative;
+  bottom: 60em;
+  left: 16em;
+}
+
+.form-paragraph {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  color: #ebebff;
+  position: relative;
+  bottom: 118em;
+  left: 25em;
+}
+
+.forms {
+  position: relative;
+  bottom: 145em;
+  left: 35em;
+  height: 50px;
+}
```

Comment:
- [ ] There is too much of width which leads to horizontal scrolling of the page. Kindly try to fix the positioning issue of the form section.

![image](https://user-images.githubusercontent.com/53019916/126491269-7d242f7a-1b0b-4693-83d0-6b83fdda9c92.png)